flink-libraries/flink-ml/src/main/scala/org/apache/flink/ml/recommendation/ALS.scala

```diff
@@ -535,8 +581,17 @@ object ALS {
     itemOut: DataSet[(Int, OutBlockInformation)],
     userIn: DataSet[(Int, InBlockInformation)],
     factors: Int,
-    lambda: Double, blockIDPartitioner: FlinkPartitioner[Int]):
+    lambda: Double, blockIDPartitioner: FlinkPartitioner[Int],
+    implicitPrefs: Boolean,
+    alpha: Double):
   DataSet[(Int, Array[Array[Double]])] = {
+    // retrieve broadcast XtX matrix in implicit case
+    val XtXtoBroadcast = if (implicitPrefs) {
```

Comment:
I'm a bit confused with the notation here, is this matrix the `YtY ` matrix from the paper? If yes, I would recommend sticking to the notation of the paper to avoid confusion.